docs/reference/ACL-migration.md

```diff
@@ -0,0 +1,69 @@
+### The new ACL model
+As mentioned in our [Security Update](https://docs.lakefs.io/posts/security_update.html#whats-changing) the coming lakeFS version introduces a simplified authorization mechanism based on ACLs.
+
+### Permissions
+In the ACL model Every group will have a permission attached to it,
+lakeFS provides the following permissions:
+
+| Permission | Allows | Existing Group |
+|------------|--------------------------------------------|---------------------------|
+| **Read** | Read operations, creating access keys. | Viewers |
+| **Write** | Allows all data read and write operations. | Developers |
+| **Super** | Allows all operations except auth. | SuperUsers (with changes) |
+| **Admin** | Allows all operations. | Admins |
+
+### Scopes
+
+When granted to a group, permissions Read, Write, and Super may be scoped to a set of repositories.
+Admin includes global abilities that apply across repos and cannot be scoped to a set of repositories
+
+### ACL
+
+You can now attach Permissions and scope them to groups in the Groups page
+There are 4 default groups, named after the 4 permissions. Each group is global (applies for all repositories).
+The group list initially would be
+
+| Group ID    | Default Permission | Created at       | Repositories |
+|-------------|--------------------|------------------|--------------|
+| **Read**    | Read               | \<Creation date> | All          |
+| **Write**   | Write              | \<Creation date> | All          |
+| **Super**   | Super              | \<Creation date> | All          |
+| **Admin**   | Admin              | \<Creation date> | All          |
+
+
+In the UI:
+
+![Groups page has a dropdown to edit permission for each
+group]( ../assets/img/ACL-groups-with-perms.png)
+
+
+
+### The migration
+
+Upgrading the lakeFS version will require running a migrating to our ACL authorization model in case:
+- One or more of the basic poicies was modified
+- New policies where defined
+- Users have attached policies
+
+In order to run the migration run:
+```
+TODO - Add the command
+```
+
+The command defaults to dry-run,
+We recommend running the script only after running the dry-run and going over the warnings
+In order to apply the migration you will need  `--yes`  flag
```

Comment:
```suggestion
The command defaults to dry-run.
Running the script and go over the warnings about widened or otherwise changed permissions.
To apply the migration, re-run with the  `--yes`  flag
```

Reply:
@arielshaqed , I don't understnad what you meant in:
> Running the script and go over the warnings about widened or otherwise changed permissions.